react/features/screenshot-capture/actions.js

```diff
@@ -30,33 +31,37 @@ function setScreenshotCapture(enabled) {
 * @param {boolean} enabled - Bool that represents the intention to start/stop screenshot captures.
 * @returns {Promise}
 */
-export function toggleScreenshotCaptureEffect(enabled: boolean) {
+export function toggleScreenshotCaptureSummary(enabled: boolean) {
     return async function(dispatch: (Object) => Object, getState: () => any) {
         const state = getState();
 
         if (state['features/screenshot-capture'].capturesEnabled !== enabled) {
             const { jitsiTrack } = getLocalVideoTrack(state['features/base/tracks']);
 
-            if (!ongoingEffect) {
-                ongoingEffect = await createScreenshotCaptureEffect(state);
+            if (!screenshotSummary) {
+                try {
+                    screenshotSummary = await createScreenshotCaptureSummary(state);
+                } catch (err) {
+                    logger.error('Cannot create screenshotCaptureSummary', err);
+                }
             }
 
             // Screenshot capture effect doesn't return a modified stream. Therefore, we don't have to
             // switch the stream at the conference level, starting/stopping the effect will suffice here.
             if (enabled) {
                 try {
-                    await ongoingEffect.startEffect(
+                    await screenshotSummary.start(
```

Comment:
You can pass the JitsiTrack directly to this function instead, and then all the metadata is already there, instead of having to separate the videoType just to check it.